### PR TITLE
jupyter-incubator/dashboards incorporation proposal

### PR DESCRIPTION
**Pull request by parente**
This PR proposes graduating the [dashboard layout extension](https://github.com/jupyter-incubator/dashboards) from jupyter-incubator. Per the roadmap and discussion at the Spring '16 dev meeting, it does **not** include the other, related projects such as the dashboards server. (We'll follow-on with a proposal for that and others.)

cc'ing folks who commented on the [original incubation proposal](https://github.com/jupyter-incubator/proposals/pull/4) to start: @vinomaster @rgbkrk @ellisonbg @fperez @Carreau @lbustelo @lmeyerov @minrk @damianavila

### CODE REVIEW

**Comment by minrk** [comment]
👍

**Comment by fperez** [comment]
While eventually I figured out what you meant, I would suggest rewording the first paragraph in [the repo README](https://github.com/jupyter-incubator/dashboards) and say something like:

```
The Jupyter Incubator Dasboards effort covers:

1. ...
2. ...
3. ...

This repository focuses on (1) above, while (2) and (3) are dealt with at ... and ...
```

I think the current language is a bit ambiguous on first read as to what is the whole effort vs what is covered by this repo.

**Comment by parente** [comment]
@fperez said:

> While eventually I figured out what you meant, I would suggest rewording the first paragraph in the repo README and say something like:

Will do. I'll double-check the proposal text too since it includes a similar statement.

> I think the current language is a bit ambiguous on first read as to what is the whole effort vs what is covered by this repo.

The curse of multiple repositories strikes again! 😱

**Comment by parente** [comment]
@fperez With @willingc 's help, we've stood up a RTD site with documentation about how to install, use, and develop the dashboards extension. 

https://jupyter-dashboards-layout.readthedocs.io

I think it addresses the concern about projects needing doc before leaving the incubator from our conversation on the 2016-07-05 stand-up. I'll update this PR to refer to the site accordingly.

**Comment by parente** [comment]
@willingc did you mean to merge the proposal?

**Comment by willingc** [comment]
@parente Nope. Too early here. I was thinking this PR was just the docs. I reverted the merge but that may have messed the history up.

FWIW, it would be great to graduate this project ;-)

Sorry for the confusion.

**Comment by parente** [comment]
@willingc No problem. Can you reopen this pull request though so I don't have to submit a new one? I don't have permissions to reopen it myself.

**Comment by rgbkrk** [comment]
I don't see a way to reopen a PR here, only a way to revert.

**Comment by parente** [comment]
> I don't see a way to reopen a PR here, only a way to revert.

I guess once you merge, you can't reopen. You can only reopen if you close without merging. Maybe.

**Comment by rgbkrk** [comment]
There might be an API way. Otherwise, we would probably need to open yet another PR with your current branch. :(

**Comment by willingc** [comment]
Sorry again :(

**Comment by parente** [comment]
@willingc Not a problem, really. New PR is #20.

**Comment by willingc** [comment]
Thanks @parente.
